### DISCUSSION TITLE
一个管理 OpenClash 的iOS App，欢迎大家提意见

### DISCUSSION THREAD

**Discussion by bin64** [vernesong/OpenClash#4243] [Show and tell]
大家好，因为我自己一直在使用OpenClash，虽然 OpenClash 提供了 Web 界面的控制面板，但那个在竖屏设备上使用并不是特别友好（主要是查看活动连接，滑动比较麻烦），我自己常常查看这些连接的信息。所以写了这个 App，可以比较方便的在 iPhone 上对 OpenClash 进行一些管理。

这个是 App 的界面：
![<img src="preview.png" alt="Clash Dash Preview" width="900">](https://raw.githubusercontent.com/bin64/Clash-Dash/refs/heads/main/preview.png)

项目地址是：https://github.com/bin64/Clash-Dash

App 已经上架了商店，还在不断更新和完善。目前实现了：
- Clash 控制器的全部功能（这个是主要的界面，因为我觉得 OpenClash 一旦设置好了之后并不需要怎么维护，主要就是使用控制器来切换一些节点、查看活动连接）。
- OpenClash 的部分功能，例如修改配置文件、切换配置文件、添加或编辑还有更新订阅。

我想的是做一个轻量的 OpenClash 管理工具，没有想着要实现在手机上操作 OpenClash 的全部功能，下一步应该会增加对规则覆写的支持。

下面是商店版本的兑换码，欢迎对这个 App 感兴趣各位兑换使用。大家也可以自行 clone 编译安装，我也提供了 .ipa 安装包，可自签或者巨魔安装。

https://redeem.ym.si/6Q9II8

![iOS 兑换码 兑换状态](https://redeem.ym.si/6Q9II8/status)

👏欢迎大家查看项目、提出建议、友好交流。

**Comment by bin64** [comment]
1.2.1 使用 OpenWRT 方式登录有 bug，已经修复提交审核了。

1.2.2 版本已支持添加自定义规则。

**Reply by Hui-131-PM** [reply]
1

**Comment by vernesong** [comment]
可以根据已有的链接添加一些插件专门的功能
https://github.com/vernesong/OpenClash/blob/dcb127c2b37e9f8ef88d34cd59dbaccd0624fc46/luci-app-openclash/luasrc/controller/openclash.lua#L20

**Reply by bin64** [reply]
感谢回复！我也是这样想的，使用 OpenWRT 登录后能做的还挺多的，就是在想怎么把这些功能整合起来，避免太过于臃肿。

如果您感兴趣，欢迎使用兑换码兑换体验：

兑换地址：https://redeem.ym.si/RP5W2Z

兑换状态：![Clash Dash iOS 兑换码 兑换状态](https://redeem.ym.si/RP5W2Z/status)
